src/environments/environment.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
   production: false,
-  ethstatsService: 'wss://baklava-ethstats.celo-testnet.org',
-  // ethstatsService: 'ws://localhost:3000',
+  // ethstatsService: 'wss://baklava-ethstats.celo-testnet.org',
+  ethstatsService: 'ws://localhost:3000',
```

Comment:
is the intended?

Reply:
Mmmm, I use it to test it
I prefer everything pointing to localhost (at least for testing)
And that `baklava-ethstats` don't exist anymore

Reply:
ok 👍